_posts/2015/2015-12-07-codemotion-madrid.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: "Codemotion 2015 - One ePagee was there!"
+date: "2015-12-07 12:00:00"
+icon: search
+categories: comments
+authors: ["Ángel García"]
+---
+The Codemotion conference took place in Madrid on November 28 and 29. With more than 1900 software enthusiasts, Codemotion is the biggest IT event in Spain. This conference was sponsored by companies such as IBM, Microsoft, Arsys, Mozilla, Google... Two days filled with talks from 12 different tracks, where each track
```

Comment:
This conference was sponsored by companies such as IBM, Microsoft, Arsys, Mozilla and Google... Two days filled with talks from 12 different tracks, where each track